src/Auth/RoleRepository.php

```diff
@@ -66,6 +60,8 @@ public function save(Role $role)
         }
 
         $this->write($roles);
+
+        Blink::forget('user-roles');
```

Comment:
question, perhaps ignorant, but if the data is cached only per request, why do you need to clear it?

Reply:
Because of this:

```php
$allRoles = $roles->all();

$newRole->save();

$roles->all(); // wont include $newRole
```

Reply:
duh, of course.